docker/dev.yml

```diff
@@ -31,16 +31,22 @@ x-robotoff-dev: &robotoff-dev
       - ./gh_pages:/opt/robotoff/gh_pages
 
 x-networks-productopener-local: &networks-productopener-local
-    # in dev we want to be able to communicatet with product opener
+    # in dev we want to be able to communicate with product opener
     # using our docker-compose environment
     networks:
-      - po_local
-      - webnet
+      po_local:
+      webnet:
 
 services:
   api:
     <<: *robotoff-dev
-    <<: *networks-productopener-local
+    networks:
+      po_local:
+        aliases:
+        # trick: make robotoff available outside and inside with same name
```

Comment:
Can you explain in more details how it works?

Reply:
I added `# with this alias, we can call it from inside with same name as outside`

An alias is an additional name you give to the container in this network. Here I give it the name "robotoff.openfoodfacts.localhost". I need this because on Product Opener I have only one config which is ROBOTOFF_URL and it is used by the javascript (so in the browser, which is not in docker space) and by the perl code (which is inside docker). So I need a name which work for both. And the easiest way is this :-)